Test failures on 32-bit systems

After patching dask (dask/dask#2019), there are a few [32-bit build failures](https://copr-be.cloud.fedoraproject.org/results/qulogic/dask/fedora-25-i386/00520683-python-distributed/build.log.gz):
```
=================================== FAILURES ===================================
______________________________ test_sparse_matrix ______________________________
    def test_sparse_matrix():
        sparse = pytest.importorskip('scipy.sparse')
        sp = sparse.eye(10)
        assert sizeof(sp.todia()) >= 152
        assert sizeof(sp.tobsr()) >= 232
>       assert sizeof(sp.tocoo()) >= 252
E       assert 240 >= 252
E        +  where 240 = sizeof(<10x10 sparse matrix of type '<type 'numpy.float64'>'\n	with 10 stored elements in COOrdinate format>)
E        +    where <10x10 sparse matrix of type '<type 'numpy.float64'>'\n	with 10 stored elements in COOrdinate format> = <bound method dia_matrix.tocoo of <10x10 sparse matrix of type '<type 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>>()
E        +      where <bound method dia_matrix.tocoo of <10x10 sparse matrix of type '<type 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>> = <10x10 sparse matrix of type '<type 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>.tocoo
distributed/tests/test_sizeof.py:46: AssertionError
```

Unfortunately not, it still fails with that change.

Odd that it doesn't fail on your 32-bit install. I even tried using a 32-bit conda install, where I get this slightly different result:
```
___________________________ test_sparse_matrix ______________________________

    def test_sparse_matrix():
        sparse = pytest.importorskip('scipy.sparse')
        sp = sparse.eye(10, dtype=np.float64)
        assert sizeof(sp.todia()) >= 152
        assert sizeof(sp.tobsr()) >= 232
>       assert sizeof(sp.tocoo()) >= 252
E       assert 248 >= 252
E        +  where 248 = sizeof(<10x10 sparse matrix of type '<class 'numpy.float64'>'\n	with 10 stored elements in COOrdinate format>)
E        +    where <10x10 sparse matrix of type '<class 'numpy.float64'>'\n	with 10 stored elements in COOrdinate format> = <bound method dia_matrix.tocoo of <10x10 sparse matrix of type '<class 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>>()
E        +      where <bound method dia_matrix.tocoo of <10x10 sparse matrix of type '<class 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>> = <10x10 sparse matrix of type '<class 'numpy.float64'>'\n	with 10 stored elements (1 diagonals) in DIAgonal format>.tocoo
```
I only used the released versions `dask`, `joblib`, or `zict` at first, but even installing them from git does not work.

--- a/distributed/tests/test_sizeof.py
+++ b/distributed/tests/test_sizeof.py
@@ -41,13 +41,13 @@ def test_pandas():
 def test_sparse_matrix():
     sparse = pytest.importorskip('scipy.sparse')
     sp = sparse.eye(10)
-    assert sizeof(sp.todia()) >= 152
+    assert sizeof(sp.todia()) >= 152  # these are the 32 bit values
     assert sizeof(sp.tobsr()) >= 232
-    assert sizeof(sp.tocoo()) >= 252
+    assert sizeof(sp.tocoo()) >= 248
     assert sizeof(sp.tocsc()) >= 232
-    assert sizeof(sp.tocsr()) >= 260
-    assert sizeof(sp.todok()) >= 260
-    assert sizeof(sp.tolil()) >= 324
+    assert sizeof(sp.tocsr()) >= 238
+    assert sizeof(sp.todok()) >= 192
+    assert sizeof(sp.tolil()) >= 210
 
 
 def test_serires_object_dtype():

